MATURITY.md

```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
+|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CS10 | The project maintains a public list of its contributors who have decision power. The project's PMC (Project Management Committee) consists of those contributors.                                                         | :white_check_mark: Committers and PPMC members are listed on the website’s community page, see [newcommitter](https://kyuubi.apache.org/become_committer.html)                                                 |
+| CS20 | Decisions require a consensus among PMC members and are documented on the project's main communications channel. The PMC takes community opinions into account, but the PMC has the final word.                           | :white_check_mark: The project has been making important decisions on the mailing lists.                                                                                                                       |
+| CS30 | The project uses documented voting rules to build consensus when discussion is not sufficient.                                                                                                                            | :white_check_mark: The project uses the standard ASF voting rules.                                                                                                                                             |
+| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_check_mark: The project has not used a veto at any point during incubation.                                                                                                                             |
+| CS50 | All "important" discussions happen asynchronously in written form on the project's main communications channel. Offline, face-to-face or private discussions that affect the project are also documented on that channel. | :white_check_mark: The project has been making important decisions on the project mailing lists. Minor decisions may occasionally happen during code reviews, which are also asynchronous and in written form. |
```

Comment:
For example:

Major features discussion:

- [[DISCUSS][KPIP-4] Support to submit batch job](https://lists.apache.org/thread/mfn69bhnrqgm67wcgvc08fbm0qwkfj99)
- [[KPIP-4] Support to submit batch job](https://github.com/apache/incubator-kyuubi/discussions/2304)

- [[DISCUSS][KPIP-5] Introduce Kyuubi Web UI](https://lists.apache.org/thread/5rx1sc7syny9bt8kpr6lnoyr4jmnhkkp)
- [[KPIP-5] Introduce Kyuubi Web UI](https://github.com/apache/incubator-kyuubi/discussions/2651)
- [[VOTE][KPIP-5] Introduce Kyuubi Web UI](https://lists.apache.org/thread/qk5ftp6jd0w37lv018mn4zx75d7m2n0w)

Deprecation discussion:

- [[DISCUSS] Spark 3.0 Support EOL Progress](https://lists.apache.org/thread/3mw8zz9r0tzg2dpj6717xy59610bhkp4)